source/_integrations/threshold.markdown

```diff
@@ -76,6 +76,26 @@ name:
   default: Threshold
 {% endconfiguration %}
 
+## Matrix of state change behavior
+
+### Sensor value rising
+```
+set             turns on when             turns off when
+---             -------------             --------------
+only upper      sensor > (upper + hyst)   never
+only lower      never                     sensor > (lower + hyst)
+upper & lower   sensor > (lower + hyst)   sensor > (upper + hyst)
+```
```

Comment:
Let's make use of tables here, instead of trying to add tables in a code fenced block.